components/ringcentral/sources/new-call-recording/new-call-recording.js

```diff
@@ -0,0 +1,46 @@
+const common = require("../common/timer-based");
+
+module.exports = {
+  ...common,
+  key: "ringcentral-new-call-recording",
+  name: "New Call Recording",
+  description: "Emits an event when a call recording is created",
+  props: {
+    ...common.props,
+    extensionId: { propDefinition: [common.props.ringcentral, "extensionId"] },
+  },
+  methods: {
+    ...common.methods,
+    generateMeta(data) {
+      const {
+        id,
+        startTime: timestamp,
+      } = data;
+
+      const summary = `New call recording`;
```

Comment:
nothing else we can include in summary here?

Reply:
We can add the following, whichever makes more sense to you:
- Session ID (i.e. what identifies the call that was recorded)
- Phone number of the client (the "from" phone number if it was an inbound call, or "to" phone number if it was an outbound call)

Reply:
Either of those is fine to me. Just something to differentiate the events from summary perspective. Maybe the phone number is more useful? But a user of the source would know best what's useful so let's start with whichever you think is better for now.